src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        String readFile = fileReader(file, fromFileName);
+        File newFile = new File(toFileName);
+
+        try {
+            newFile.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t create file", e);
+        }
+        fileWriter(newFile, getResult(readFile));
+    }
+
+    private String fileReader(File file, String fileName) {
```

Comment:
method required redundant info, both file and fileName, you can get one from the other. Delete `file` from the arguments